docky/common/project.py

```diff
@@ -13,92 +13,46 @@ class Project(object):
-            # service.labels() do not contain docky.main.service
-            # see also compose.service.merge_labels
-            if labels:
-                if labels.get("docky.main.service"):
-                    return service
-
-    def get_containers(self, service=None):
-        kwargs = {}
-        if service:
-            kwargs["services"] = [service]
-        return docker.compose.ps(**kwargs)
+            if labels and labels.get("docky.main.service"):
+                return service
 
     def display_service_tooltip(self):
-        infos = self._get_services_info()
-        for service in self.project.get("services"):
-            dict_service = self.project["services"].get(service)
-            labels = dict_service.get("labels", {})
-            if labels.get("docky.access.help"):
-                # TODO remove after some versions
-                logger.warning(
-                    "'docky.access.help' is replaced by 'docky.help'. "
-                    "Please update this key in your docker files.")
-            if infos.get(dict_service.get("name")):
-                # some applications provide extra parameters to access resource
-                infos[service.name] += labels.get("docky.url_suffix", "")
-                logger.info(infos[service.name])
-            if labels.get("docky.help"):
-                logger.info(labels.get("docky.help"))
-
-    def _get_services_info(self):
-        """ Search IP and Port for each services
-        """
-        infos = {}
-        main_service = self._get_main_service(self.project)
-        for service in self.project.get("services"):
-            if service != main_service:
-                continue
-            serv = self.project["services"][service]
-            proj_key = [
-                x for x in serv["networks"].keys()]
-            proj_key = proj_key and proj_key[0] or False
-            if not serv["networks"]:
-                continue
-            ip = serv["networks"].get("IPAdress", "")
-            info = {
-                "name": serv["labels"].get(
-                    "com.docker.compose.service", ""),
-                "ip": ip,
-                "port": [x for x in serv.get("ports", "")],
-            }
-            if info["name"] != "db" and info.get("port"):
-                urls = ["http://%s:%s" % (info["ip"], port.replace("/tcp", ""))
-                        for port in info["port"][0]]
-                # There is no web app to access 'db' service: try adminer for that
-                infos[info["name"]] = "%s %s" % (info["name"], " ".join(urls))
-        return infos
+        for _name, service in self.project.get("services").items():
+            docky_help = service.get("labels", {}).get("docky.help")
+            if docky_help:
+                logger.info(docky_help)
 
     def create_volume(self):
         """Mkdir volumes if they don't exist yet.
 
         Only apply to external volumes.
         docker-compose up do not attemps to create it
```

Comment:
```suggestion
        docker compose up create volumes automatically, but docker compose run don't
```

Reply:
in fact the comment was totally wrong docker compose always create the volume but with the user root. I  fix it